packages/client/src/app/components/card/CardItem.vue

```diff
@@ -0,0 +1,23 @@
+<script setup>
+const props = defineProps({
+  clickable: Boolean,
+  title: String,
+  text: String
+})
+</script>
+
+<template>
+  <div class="card" :class="{ 'card-clickable': props.clickable }">
+    <div class="card-body">
+      <h5 class="card-title fw-bold">{{ props.title }}</h5>
```

Comment:
For accessibility, headers should only increase by one (so starting from h1, then h2 etc). Maybe it's OK to override the h2 and h3 sizes in these views only?

Reply:
The thing is that this component will be probably reused on another page where we might have already defined a h1 to h3 for example and If I put h1 or h2 here it might look weird for other pages.

Reply:
That's OK, it is a best practice and not a requirement so we can decide to keep it this way.